ansible/roles/cartodb.app/tasks/main.yml

```diff
@@ -0,0 +1,51 @@
+---
+
+ - name: Precompile static assets
+   command: bundle exec grunt --environment {{ cartodb_environment }}
+   sudo: False
+   args:
+     chdir: "{{ cartodb_dir }}"
+
+ # TODO: Template if necessary
+ - name: Create app_config.yml
+   command: cp config/app_config.yml.sample config/app_config.yml
```

Comment:
It might be good to add a `creates=` specification here so that this doesn't clobber `app_config.yml` on consecutive runs; same for the `database.yml` one below.